docs/en/faq.md

```diff
@@ -123,6 +123,64 @@ You can accelerate report loading by converting QuickSight datasets to SPICE mod
     1. Enabling SPICE will result in charges based on the amount of space used. Pricing details can be found in the QuickSight pricing page. If the dashboard is frequently viewed, enabling SPICE can reduce the access load on the Redshift database, but it may also increase data latency.
     2. By default, the solution refreshes data in SPICE using an incremental update method. The refresh process is scheduled at 6 AM daily in your dashboard's time zone. You can manually adjust the update schedule in QuickSight.
 
+
+### How to implement a dedicated Redshift for Analytics Studio?
+
+Redshift supports [sharing data across different Redshift clusters][redshift-share-data], allowing you to use a dedicated Redshift cluster for Analytics Studio to achieve better query performance and cost optimization.
+
+Before implementing Amazon Redshift data sharing, please note the following:
+
+- You can share data between cluster types, as well as between configured clusters and serverless ones.
+- Only **RA3** type clusters and **Redshift serverless** support data sharing.
+
+Taking Redshift serverless as an example of data sharing, follow these operational steps:
+
+1. [Create a Redshift serverless][serverless-console-workflows] as the data consumer.
+2. Run SQL in the producer Redshift database (The project database configured in the Clickstream solution) to create a data share and grant consumer permissions:
+    ```sql
+    -- Create Datashare
+
+    CREATE DATASHARE bi SET PUBLICACCESSIBLE FALSE;
+    ALTER DATASHARE bi ADD SCHEMA <schema>;
+    ALTER DATASHARE bi ADD ALL TABLES IN SCHEMA <schema>;
+
+    -- Grant the Datashare to the consumer Redshift.
+
+    GRANT USAGE ON DATASHARE bi TO NAMESPACE '<target namespace id>';
+    ```
+    Replace `<schema>` with the schema you want to share, and `<target namespace id>` with the consumer Redshift serverless namespace ID.
```

Comment:
user "consumer" instead of "target"?

Reply:
better explain to user what is the 'bi', can customer user other name for database? if yes, should be <database name>